source/org/zfin/marker/service/MarkerService.java

```diff
@@ -862,6 +862,26 @@ public static boolean markerHasAlias(Marker marker, String alias) {
         return false;
     }
 
+
+    public static MarkerAlias createMarkerAlias(Marker marker, String newAlias, List<String> publicationIDs) {
+        Iterator<String> pubIterator = publicationIDs.iterator();
```

Comment:
I would write in one line:
String pubID = publicationIDs.iterator().next();
Then you don't have deal with the iterator which has not a strong meaning as pubID.